python/tvm/relay/expr.py

```diff
@@ -172,6 +172,20 @@ def name_hint(self):
         name = self.vid.name_hint
         return name
 
+    def __call__(self, *args):
```

Comment:
Much of the ADT code assumed that the call operator for vars was overridden, so I added this to make those work. Is there a general position on operator overriding in the Python bindings? Is an override like this alright to have?